src/panel-app/ErrorBoundary.component.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export default class ErrorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { caughtError: null };
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { caughtError: error };
+  }
+
+  render() {
+    if (this.state.caughtError) {
+      // You can render any custom fallback UI
+      return (
+        <div
+          style={{
+            padding: "16px"
+          }}
+        >
+          <h1>single-spa Inspector broke!</h1>
+          <p>
+            Please close and repopen the devtools to get this working again.
+          </p>
+          <p>
+            Also, report this error{" "}
+            <a href="https://github.com/CanopyTax/single-spa-inspector/issues/new">
```

Comment:
It'd be pretty nice to automatically add some data to the URL which gets populated into the issue. See https://help.github.com/en/articles/about-automation-for-issues-and-pull-requests-with-query-parameters